blueocean-personalization/package.json

```diff
@@ -9,7 +9,8 @@
     "test": "gulp test",
     "test:watch": "gulp test:watch",
     "bundle": "gulp bundle",
-    "bundle:watch": "gulp bundle:watch"
+    "bundle:watch": "gulp bundle:watch",
+    "postinstall": "node ../bin/cleanshrink.js"
```

Comment:
yes not sure if front end plugin does that. I think I pinged @tfennelly about that once as being not a bad idea. But in general, post install is not great, if there is any other way to do it.